lab_2_retrieval_w_bm25/start.py

```diff
@@ -26,7 +28,18 @@ def main() -> None:
             documents.append(file.read())
     with open("assets/stopwords.txt", "r", encoding="utf-8") as file:
         stopwords = file.read().split("\n")
-    result = None
+    text_1 = remove_stopwords(tokenize('''There was a boy, who was a wizard. He used his wand to do
```

Comment:
Why do you use this sentences? Use files

Reply:
Now there are files in start